symengine/lib/symengine_wrapper.pyx

```diff
@@ -2740,6 +2788,32 @@ def exp(x):
     cdef Basic X = sympify(x)
     return c2py(symengine.exp(X.thisptr))
 
+def isqrt(n):
+    cdef Basic _n = sympify(n)
+    require(_n, Integer)
+    return c2py(<RCP[const symengine.Basic]>(symengine.isqrt(deref(symengine.rcp_static_cast_Integer(_n.thisptr)))))
+
+def perfect_power(n):
+    cdef Basic _n = sympify(n)
+    require(_n, Integer)
+    return symengine.perfect_power(deref(symengine.rcp_static_cast_Integer(_n.thisptr)))
+
+def perfect_square(n):
```

Comment:
I would rename this to `is_square` instead of aliasing. Two names for the same method is needed only for sympy compatibility or backwards compatibility.